tests/acceptance/TestHelpers/SettingsHelper.php

```diff
@@ -32,6 +32,39 @@
 class SettingsHelper {
 	private static string $settingsEndpoint = '/api/v0/settings/';
 
+	private const BUNDLE_ID = "2a506de7-99bd-4f0d-994e-c38e72c28fd9";
+
+	private const DEFAULT_SETTING_EVENTS = [
+		'Disable Notifications' => '33ffb5d6-cd07-4dc0-afb0-84f7559ae438',
+		'Auto Accept Shares' => 'ec3ed4a3-3946-4efc-8f9f-76d38b12d3a9',
+		'File Rejected' => 'fe0a3011-d886-49c8-b797-33d02fa426ef',
```

Comment:
Let's use the same `displayName` we get in the response of bundles list.
```suggestion
		'File rejected' => 'fe0a3011-d886-49c8-b797-33d02fa426ef',
```